adafruit_hid/keyboard.py

```diff
@@ -143,3 +148,12 @@ def _remove_keycode_from_report(self, keycode):
             for i in range(_MAX_KEYPRESSES):
                 if self.report_keys[i] == keycode:
                     self.report_keys[i] = 0
+
+    @property
+    def led_status(self):
+        """Returns the last received report"""
+        return self._keyboard_device.last_received_report
+
+    def led_on(self, led_code):
+        """Returns whether an LED is on based on the led code"""
```

Comment:
Could you include an example, using one or more of the constants?

Reply:
I have added an example using LED_CAPS_LOCK.